# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/layers/LayersTypes.h
# Commit: 16c5dd2cfd60
# Full Hash: 16c5dd2cfd600a62c80ca17cfb35c0868e016056
# Author: Jeff Gilbert <jgilbert@mozilla.com>
# Date: 2020-06-15 21:48:38
# Regressor Bug: 1632249
# File Overlap Count: 3
# Description:
#   Bug 1632249 - Support compositing out-of-process WebGL. r=handyman,lsalzman,nical,geckoview-reviewers,agi,imanol
#   
#   * Majorly simplity CanvasRenderer
#   * Replace GLScreenBuffer with trivial GLSwapChain
#   * Use descriptor structs so that future SharedSurface changes aren't so painful
# ==============================================================================

diff -r ca8139d58d58 -r 16c5dd2cfd60 gfx/layers/LayersTypes.h
--- a/gfx/layers/LayersTypes.h	Fri Jun 12 21:53:14 2020 +0000
+++ b/gfx/layers/LayersTypes.h	Mon Jun 15 18:25:55 2020 +0000
@@ -167,6 +167,7 @@
   MacIOSurface,
   AndroidNativeWindow,
   WaylandDMABUF,
+  EGLImage,
   Last
 };
 